File: Basics_with_Python/pythonBasicBook/!PythonBasicCourse/3_Conditional Statements Advanced/8_Conditional_Statements_Advanced - Exercise - Lab/more_exercises/06. Truck Driver.py
season = input()
km_monthly = float(input())
km_price = 0
if km_monthly <= 5000:
    if season == "Spring" or season == "Autumn":
        km_price = 0.75
    elif season == "Summer":
        km_price = 0.90
    elif season == "Winter":
        km_price = 1.05
elif 5000 < km_monthly <= 10000:
    if season == "Spring" or season == "Autumn":
        km_price = 0.95
    elif season == "Summer":
        km_price = 1.10
    elif season == "Winter":
        km_price = 1.25
elif 10000 < km_monthly <= 20000:
    if season == "Spring" or season == "Autumn" or season == "Summer" or season == "Winter":
        km_price = 1.45
print(f"{(km_price * km_monthly * 4 * 0.9):.2f}")

File: Basics_with_JavaScript/78_Conditional_Statements_Advanced/More_Exersises/06._Truck_Driver.js
function vladi(input){

    let season = input[0]
    let distancePerMonth = Number(input[1])
    let pricePerKm = 0

    if (distancePerMonth <= 5000) {
        switch (season) {
            case "Spring":
            case "Autumn":
                pricePerKm = 0.75;
                break;
            case "Summer":
                pricePerKm = 0.90;
                break;
            case "Winter":
                pricePerKm = 1.05;
                break;
            default:
                break;
        }
    } else if (distancePerMonth > 5000 && distancePerMonth <= 10000) {
        switch (season) {
            case "Spring":
            case "Autumn":
                pricePerKm = 0.95;
                break;
            case "Summer":
                pricePerKm = 1.10;
                break;
            case "Winter":
                pricePerKm = 1.25;
                break;
            default:
                break;
        }
    } else if (distancePerMonth > 10000 && distancePerMonth <= 20000) {
        pricePerKm = 1.45;
    }
    let salary = ((4 * distancePerMonth * pricePerKm)* 0.9).toFixed(2)
    console.log(salary);
    }
   
    vladi(["Summer", "3455"]);
    vladi(["Winter", "4350"]);
    vladi(["Spring", "1600"]);
    vladi(["Winter", "5678"]);
    vladi(["Autumn", "8600"]);
    vladi(["Winter", "16042"]);
    vladi(["Spring", "16942"]);